osdep/mac/app_bridge.m

```diff
@@ -139,4 +140,9 @@ int cocoa_main(int argc, char *argv[])
 {
     return [(Application *)[Application sharedApplication] main:argc :argv];
 }
-
+#else
+int cocoa_main(int argc, char *argv[])
```

Comment:
Is this stub function still needed after meson change? Shouldn't be used now, no?

Reply:
of course you are right. removed it, thanks.